File: refrigitz 12/TakeImage.cpp
#include "TakeImage.h"


namespace GalleryStudio
{

	TakeImage::TakeImage(int Senderint, Customers *SenderCustomer)
	{
		InitializeInstanceFields();
		if (Senderint != 0)
		{
			if (SenderCustomer != nullptr)
			{
				Day = DateTime::Today.Day;
				Month = DateTime::Today.Month;
				Year = DateTime::Today.Year;
				PhotoNumber = Senderint;
				CustomerObject = SenderCustomer;
			}
		}
	}

	const int &TakeImage::getDayAccess() const
	{
		return Day;
	}

	void TakeImage::setDayAccess(const int &value)
	{
		Day = value;
	}

	const int &TakeImage::getMonthAccess() const
	{
		return Month;
	}

	void TakeImage::setMonthAccess(const int &value)
	{
		Month = value;
	}

	const int &TakeImage::getYearAccess() const
	{
		return Year;
	}

	void TakeImage::setYearAccess(const int &value)
	{
		Year = value;
	}

	const int &TakeImage::getPhotoNumberAccess() const
	{
		return PhotoNumber;
	}

	void TakeImage::setPhotoNumberAccess(const int &value)
	{
		PhotoNumber = value;
	}

	Customers *TakeImage::getCustomersAcess() const
	{
		return CustomerObject;
	}

	void TakeImage::setCustomersAcess(Customers *value)
	{
		CustomerObject = value;
	}

	void TakeImage::InitializeInstanceFields()
	{
		Day = int();
		Month = int();
		Year = int();
		PhotoNumber = int();
		CustomerObject = new Customers();
	}
}

File: GalleryStudio/TakeImage.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GalleryStudio
{
    [Serializable]
    public class TakeImage
    {
        int Day = new int();
        int Month = new int();
        int Year = new int();

        int     PhotoNumber    = new int();

        Customers CustomerObject = new Customers();
        
        public TakeImage(int Senderint,Customers SenderCustomer)
        {
            if(Senderint!=0)
                if (SenderCustomer != null)
                {
                    Day = DateTime.Today.Day;
                    Month = DateTime.Today.Month;
                    Year = DateTime.Today.Year;
                    PhotoNumber = Senderint;
                    CustomerObject = SenderCustomer;
                }
        }
        public int DayAccess
        {
            get
            {
                return Day;
            }
            set 
            {
                Day = value;
            }
        }
        public int MonthAccess
        {
            get 
            {
                return Month;
            }
            set
            {
                Month = value;
            }
        }
        public int YearAccess
        {
            get 
            {
                return Year;
            }
            set 
            {
                Year = value;
            }
        }
        public int PhotoNumberAccess
        {
            get 
            {
                return PhotoNumber;
            }
            set 
            {
                PhotoNumber = value;
            }
        }
        public Customers CustomersAcess
        {
            get
            {
                return CustomerObject;
            }
            set 
            {
                CustomerObject = value;
            }
        }
    }
}
